src/settings.rs

```diff
@@ -289,8 +290,8 @@ impl CommonSettings {
 // Builder Pattern
 impl CommonSettings {
     /// Number of build users to create
-    pub fn daemon_user_count(&mut self, count: usize) -> &mut Self {
-        self.daemon_user_count = count;
+    pub fn nix_user_count(&mut self, count: usize) -> &mut Self {
```

Comment:
Do we want to rename this to `nix_build_user_count`?

Reply:
I think so!